packages/create-amplify/src/npm_package_manager_controller.ts

```diff
@@ -28,9 +30,27 @@ export class NpmPackageManagerController implements PackageManagerController {
     if (type === 'dev') {
       args.push('--save-dev');
     }
-    await this.execa(this.executableName, args, {
-      stdio: 'inherit',
-      cwd: this.projectRoot,
-    });
+
+    let aggregatedStdout = '';
+    const aggregatorStream = new stream.Writable();
+    aggregatorStream._write = function (chunk, encoding, done) {
+      aggregatedStdout += chunk;
+      done();
+    };
+
+    try {
+      const childProcess = this.execa(this.executableName, args, {
+        stdin: 'inherit',
+        cwd: this.projectRoot,
+      });
+
+      childProcess?.stdout?.pipe(aggregatorStream);
+      childProcess?.stderr?.pipe(aggregatorStream);
+
+      await childProcess;
+      logger.debug(aggregatedStdout);
+    } catch {
+      logger.debug(aggregatedStdout);
+    }
```

Comment:
this is gonna eat error and not fail fast.
we should re-throw here.

Reply:
Addressed as part of [0735be9](https://github.com/aws-amplify/amplify-backend/pull/595/commits/0735be9988ffbeba2b82c2c3be25e57a1adae09e)